common/model-views.cpp

```diff
@@ -5098,14 +5117,17 @@ namespace rs2
         const ImVec2 icons_size{ 20, 20 };
         //TODO: Change this once we have support for loading jsons with more data than only advanced controls
         bool is_streaming = std::any_of(subdevices.begin(), subdevices.end(), [](const std::shared_ptr<subdevice_model>& sm) { return sm->streaming; });
-        const int buttons_flags = dev.is<advanced_mode>() ? 0 : ImGuiButtonFlags_Disabled;
+        const int buttons_flags = dev.is<serializable_device>() ? 0 : ImGuiButtonFlags_Disabled;
         static bool require_advanced_mode_enable_prompt = false;
         auto advanced_dev = dev.as<advanced_mode>();
-        bool is_advanced_mode_enabled = false;
+        bool is_advanced_mode_enabled = true;
         if (advanced_dev)
         {
             is_advanced_mode_enabled = advanced_dev.is_enabled();
         }
+
+        auto serializable_dev = dev.is<serializable_device>();
```

Comment:
advanced_dev can be used for this, unless the intent was that advanced does not imply serializable. If the latter is true, then perhaps they shouldn't be derived from one another?

Reply:
Also, I see that (line 5120) it can be used before, too -- maybe move up?